powerbi-docs/guidance/powerbi-modeling-guidance-for-power-platform.md

```diff
@@ -144,7 +144,7 @@ For more information on how you can achieve query folding, see [Power Query quer
 
 By default, when you use Power Query to load a Dataverse table, it retrieves all rows and all columns. When you query a system user table, for example, it could contain more than 1,000 columns. The columns in the metadata include relationships to other entities and lookups to option labels, so the total number of columns grows with the complexity of the Dataverse table.
 
-Attempting to retrieve data from all columns is an anti-pattern. It often results in extended data refresh operations, and it will cause the query to fail when the data volume exceeds 80 MB.
+Attempting to retrieve data from all columns is an anti-pattern. It often results in extended data refresh operations, and it will cause the query to fail when the time needed to return the data exceeds ten minutes.
```

Comment:
```suggestion
Attempting to retrieve data from all columns is an anti-pattern. It often results in extended data refresh operations, and it will cause the query to fail when the time needed to return the data exceeds 10 minutes.
```
Edit per Microsoft style